src/Http/Controllers/Dominion/SearchController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace OpenDominion\Http\Controllers\Dominion;
+
+use OpenDominion\Calculators\Dominion\LandCalculator;
+use OpenDominion\Calculators\Dominion\RangeCalculator;
+use OpenDominion\Calculators\NetworthCalculator;
+use OpenDominion\Models\Dominion;
+use OpenDominion\Services\Dominion\GuardMembershipService;
+use OpenDominion\Services\Dominion\ProtectionService;
+
+class SearchController extends AbstractDominionController
+{
+    public function getSearch()
+    {
+        $guardMembershipService = app(GuardMembershipService::class);
+        $landCalculator = app(LandCalculator::class);
+        $networthCalculator = app(NetworthCalculator::class);
+        $protectionService = app(ProtectionService::class);
+        $rangeCalculator = app(RangeCalculator::class);
+
+        $dominion = $this->getSelectedDominion();
+        $dominions = Dominion::with(['race.units', 'round'])
```

Comment:
Lots of queries on the page, need to eager load some more stuff. Probably race perks and race unit perks, maybe also realm